### ISSUE TITLE
Success action to same destination not working 

### ISSUE THREAD

**Issue by CyberMarsl** [mautic/mautic#9056]
## Bug Description
Checking a custom contact field for different values (i.e.: run, stay, go) which lead to the same action "set tag" sometimes work.
I put an example in the latest Mautibox.

In the form "Test - Custom Value Action" you can enter the action "run, go or stay" as text or any other text.
If you check the customfield first for the value "go", if not true, then check for "run", if not true then check for "stay", 
If in any of the cases just mentioned true, the other values are not checked, but positives are leading to "set tag avail_action".
if no match it goes to set another tag "not_avail". This works for all negatives and for the last check to be positive. on the others to be positive, it stops.

I set the route for every customvalue found to the same action (set tag avail_action). 
So what it should do is, to set the tag "avail_action", no matter which of the three tags. It only works on the last tag to continue to action.

I am not sure if it is only behaves for the "custom_value" or other too. 

A provisorial workaround seems to be to not set the action arrows, but to use the action jump.

### Steps to reproduce
1. Create a custom value for contacts
2. Create a form to get input to this custom value
3. Create a campaing, testing the custom value for the values "run", "go", "stay" (or any other you prefer)
4. Concat the testings in the campain, that the next value is only tested, if the former test was negative
5. Most Important Part: Link all the positives of the tests to THE SAME Action: Set Tag "available"
6. Link the last negative link in the testing chain to: Set Tag "not available"
6. Try differnt values for the custom value field (run, go, stay and others).

Every contact should get the tag "available" if it matches run, go or stay. Else it should have the tag "not available".
This works with "not available" and for the last positive in the chain. It does not work for the other custom value compares.

I alredy put it into mautibox staging branch, for you, so 
 
UPDATE: this is issue is not limited to cutom contacts, it also behaves the same with "if page visited" decision. I guess it is a GENERAL PROBLEM and most or all other decisions are concerned.

**Comment by adirbnology** [comment]
**this is critical issue** as building the campaign wont give the expected result and some events my behave as expected while some wont. to clarify issue occurs when u connect to events into one with the builder. by using jump to event there is no issue.

issue:
![image](https://user-images.githubusercontent.com/67871785/90042616-97ba4a80-dcd3-11ea-96bf-052fd2a5c417.png)

workaround:
![image](https://user-images.githubusercontent.com/67871785/90043148-5e360f00-dcd4-11ea-880b-13d807ac511d.png)

**Comment by RCheesley** [comment]
Thanks for raising the bug and for the workaround folks!

**Comment by patrykgruszka** [comment]
I've just got the same issue and here is what I found. If you look at database table `campaign_events` you will notice, that single campaign event supports only one path entry (entity has only one `parent_id`, `trigger_mode`, `decisoin_path` etc.). So the problem is the builder, which should not allow multiple event entries. At this moment every new entry overrides previous path and sets new decision path.

![image](https://user-images.githubusercontent.com/8580942/98543366-b3150180-2292-11eb-95f8-8021e2a2acdb.png)
![image](https://user-images.githubusercontent.com/8580942/98543239-70ebc000-2292-11eb-856a-6402164fe810.png)
